ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/Unique_Value_Renderer/README.md

```diff
@@ -1,11 +1,11 @@
-# UniqueValueRenderer
+# Unique value renderer
 
 This sample demonstrates how to use a `UniqueValueRenderer` to style different Features in a `FeatureLayer` with different Symbols.
 
 ![](screenshot.png)
 
 ## How it works
-First a `ServiceFeatureTable` and a `FeatureLayer` are constructed and added to the Map. Then a `UniqueValueRenderer` is created, and the field name to be used as the renderer field is set as ("STATE_ABBR"). You can use multiple fields; this sample only uses one. Multiple `SimpleFillSymbol`s are defined for each type of feature we want to render differently (in this case different states of the USA). `SimpleFillSymbol`s can be applied to polygon features; these are the types of features found in the feature service used for this `ServiceFeatureTable`. A default symbol is also created; this will be used for all other features that do not match the all of the `UniqueValue`s defined. Separate `UniqueValue`s objects are created which define the values in the renderer field and what symbol should be used for matching features. These are added to the `UniqueValue` list. The renderer is set on the layer and is rendered in the `MapView` accordingly.
+This sample demonstrates how to use a `UniqueValueRenderer` to style different features in a `FeatureLayer`. Using the `UniqueValueRenderer` allows for seperate symbols to be used for features that have a specific value in a certain field. In this case, the field is state abbreviations in the USA. Multiple fields can be used; this sample only uses one.
```

Comment:
Why was this description shortened? The earlier one had explained with each and every API feature used.  Were they not being used the way it was described?